src/components/screens/software-registry-screen.tsx

```diff
@@ -186,163 +193,148 @@ const SoftwareScreen = () => {
   }
 
   return (
-    <Container>
-      <Grid container direction="column" alignItems="center" mt={4}>
-        <Typography variant="h2" m={4}>
-          {strings.softwareRegistry.applications}
-        </Typography>
-        {recommendedApplications.length > 0 && (
-          <Grid item container justifyContent="center" alignItems="center" mb={4}>
-            <Typography variant="subtitle1">
-              {strings.softwareRegistry.recommendationMessage.replace(
-                "{recommendationCount}",
-                recommendedApplications.length.toString()
-              )}
-            </Typography>
-            <IconButton
-              onClick={scrollToRecommendations}
-              sx={{
-                color: "#F9473B",
-                backgroundColor: "transparent",
-                ":hover": {
-                  backgroundColor: "transparent",
-                  boxShadow: "none",
-                },
-              }}
-            >
-              <ArrowDownwardIcon />
-            </IconButton>
-          </Grid>
-        )}
-        <Grid item container justifyContent="space-between" alignItems="center">
-          <Typography variant="h3">
-            {strings.softwareRegistry.myApplications}
+    <ThemeProvider theme={SoftwareScreenTheme}>
```

Comment:
But do we really need the same provider again here? Can't this inherit from the previous?